index.js

```diff
@@ -32,6 +39,22 @@ module.exports = {
   env: {
     browser: true,
     node: true,
+    // we are using optional chaining and nullish coalescing
+    es2020: true,
+  },
+  globals: {
+    // these globals help IDEs from displaying these as undefined
+    localStorage: true,
+    process: false,
```

Comment:
why disable `process` ?

Reply:
so are you saying this should be `"writable"` not `false`?

Reply:
I'm not up to date on the latest configuration standards. If you see something that needs changing, please do so. These were notably absent so look at this like me pointing that out, not necessarily proposing the exact way to fix it.